### DISCUSSION TITLE
SB Lern Release 5.2

### DISCUSSION THREAD

**Discussion by rhwarrier** [Sunbird-Lern/Community#59] [Announcements]
[Click here](https://project-sunbird.atlassian.net/issues/?filter=12735) to see the list of issues planned for SB Lern

The following are the planned release dates of release 5.2:

Design Discussions : 16 Jan '23 - 27 Jan '23 (2 weeks)
Sprint 01 : 30 Jan '23 - 17 Feb '23 (3 weeks)
Sprint 02 : 20 Feb '23 - 10 Mar '23 (3 weeks)
Regression & Testing : 13 March '23 to '24 March (2 weeks)
Production Release : 27 March '23
Bug Fixes & Support : 28 March ' 23 to 31 March '23 (1 week)

cc : @Sunbird-Lern/sunbirdlern_workinggroup

**Comment by Krishnaj20** [comment]
@rhwarrier @reshmi-nair - is lern 5.1.0 released? We cant find the release notes in the microsite.

@vinukumar-vs

**Reply by reshmi-nair** [reply]
@Krishnaj20 It is not yet released. We will update you after the release.

**Reply by Krishnaj20** [reply]
What is the release date? It will help us plan whether optional material changes can be taken up in ED 5.1.2 sprint, which started this Monday.

**Comment by Namratakushwah** [comment]
Hello All,

Please find the Sunbird Lern Building block 5.2 sign-off here.

Environment: [[[Dev lern](https://dev.lern.sunbird.org/)]
Testing Status: Completed
Defects Dashboard: [[Link](https://project-sunbird.atlassian.net/issues/?filter=12758&jql=project%20%3D%20LR%20AND%20issuetype%20in%20(Documentation-Issue%2C%20Epic%2C%20Installation-Issues%2C%20Minor-Enhancement%2C%20RFC)%20AND%20Sprint%20in%20(276%2C%20264)%20order%20by%20created%20DESC)]

Execution Sheet: [[Link](https://docs.google.com/spreadsheets/d/1vlf59ce7Dow4HPhtBwz7IudVP8OcWQ63tdU4_efJg34/edit#gid=0)]
Scope : [[Link](https://project-sunbird.atlassian.net/issues/?filter=12758&jql=project%20%3D%20LR%20AND%20issuetype%20in%20(Documentation-Issue%2C%20Epic%2C%20Installation-Issues%2C%20Minor-Enhancement%2C%20RFC)%20AND%20Sprint%20in%20(276%2C%20264)%20order%20by%20created%20DESC)]
Building block sign-off template: [[Link](https://docs.google.com/spreadsheets/d/1tU0kfBGGXvo8Yc5hqIil4GQKtWkH1M1EjrGkfWfxo3Y/edit#gid=0)]

cc: @rhwarrier @reshmi-nair @chitranshu-keshav @Naveenkumar739 @ashokreddy1208 @poornima.ms@qualitrix.com
